robusta_krr/core/models/config.py

```diff
@@ -54,6 +55,13 @@ def validate_namespaces(cls, v: Union[list[str], Literal["*"]]) -> Union[list[st
 
         return v
 
+    @pd.validator("resources")
+    def validate_resources(cls, v: Union[list[str], Literal["*"]]) -> Union[list[str], Literal["*"]]:
+        if v == []:
+            return "*"
+
+        return v
```

Comment:
Maybe to also make those resources case-insensitive by adding `return [res.lower() for res in v]`.
With current implementation `"Deployment"` passed from user in CLI will not make it to show deployments.